frontend/locales/fr.ts

```diff
@@ -1,7 +1,864 @@
+            ProjectRename: 'Le projet a été renommé',
+            ProjectFlagged: 'Le projet a été signalé',
+            ProjectSettingsChanged: 'Les paramètres du projet ont été changés',
+            ProjectIconChanged: "L'icône du projet a été changée",
+            ProjectFlagResolved: "L'alerte a été résolue",
+            ProjectChannelCreated: 'Un canal de projet a été créé',
+            ProjectChannelEdited: 'Un canal de projet a été édité',
+            ProjectChannelDeleted: 'Un canal de projet a été supprimé',
+            ProjectInvitesSent: 'Les invitations au projet furent envoyées',
+            ProjectInviteDeclined: 'Une invitation au projet fut déclinée',
+            ProjectInviteUnaccepted: "Une invitation au projet n'a pas été acceptée",
+            ProjectMemberAdded: 'Un membre a été ajouté au projet',
+            ProjectMembersRemoved: 'Les membres du projet furent supprimés',
+            ProjectMemberRolesChanged: 'Les membres du projet ont vu leurs rôles mis à jour',
+            ProjectPageCreated: 'Une page de projet a été créée',
+            ProjectPageDeleted: 'Une page de projet a été supprimée',
+            ProjectPageEdited: 'Une page de projet a été éditée',
+            VersionVisibilityChanged: "L'état de visibilité de la version a changé",
+            VersionDeleted: 'La version a été supprimée',
+            VersionCreated: 'Une nouvelle version a été téléversée',
+            VersionDescriptionEdited: 'La description de la version a été modifiée',
+            VersionReviewStateChanged: "Le status de l'examen de la version a changé",
+            VersionPluginDependencyAdded: 'Une dépendance du plugin a été ajoutée',
+            VersionPluginDependencyEdited: 'Une dépendance du plugin a été éditée',
+            VersionPluginDependencyRemoved: 'Une dépendance du plugin a été supprimée',
+            VersionPlatformDependencyAdded: 'Une dépendance de plateforme a été ajoutée',
+            VersionPlatformDependencyRemoved: 'Une dépendance de plateforme a été supprimée',
+            UserTaglineChanged: "Le slogan de l'utilisateur a changé",
+            UserLocked: 'Cet utilisateur est verrouillé',
+            UserUnlocked: 'Cet utilisateur est débloqué',
+            UserApikeyCreated: 'Un clef api a été créée',
+            UserApikeyDeleted: 'Un clef api a été supprimée',
+            OrganizationInvitesSent: "Les invitations de l'organisation furent envoyées",
+            OrganizationInviteDeclined: "Une invitation de l'organisation fut déclinée",
+            OrganizationInviteUnaccepted: "Une invitation de l'organisation n'a pas été acceptée",
+            OrganizationMemberAdded: "Un membre fut ajouté à l'organisation",
+            OrganizationMembersRemoved: "Les membres de l'organisation furent retirés",
+            OrganizationMemberRolesChanged: "Les membres de l'organisation ont vu leurs rôles mis à jour",
```

Comment:
```suggestion
            OrganizationInvitesSent: "Les invitations de l'organisation ont été envoyées",
            OrganizationInviteDeclined: "Une invitation de l'organisation a été déclinée",
            OrganizationInviteUnaccepted: "Une invitation de l'organisation n'a pas été acceptée",
            OrganizationMemberAdded: "Un membre a été ajouté à l'organisation",
            OrganizationMembersRemoved: "Les membres de l'organisation ont été retirés",
            OrganizationMemberRolesChanged: "Les membres de l'organisation ont vu leurs rôles mis à jour",
```
Strange alternation with the *passé simple*.